en/orm/database-basics.rst

```diff
@@ -559,6 +559,13 @@ used::
             ];
         }
 
+The ``$definition`` data passed to ``convertColumnDefinition()`` could contain
+the following keys:
+
+- ``length`` The length of a column.
+- ``precision`` The precision of the column if any.
+- ``scale`` Can be included for SQLServer connections.
+
```

Comment:
I think we should guarantee the shared keys are set and if there is no value available, it is set to null.

"could contain" is too vague for something so standard, IMO.

Reply:
We would need to rework the code a bit to make those guarantees.

Reply:
Do you feel differently? I'm arguing this point because it's literally the Type definition code which we'd theoretically use for everything.

If we make it release implementation specific, it's going to be difficult.

Reply:
I agree with you, the code just needs to be updated for us to be able to give a list of always present keys.

Reply:
Do you want me to update the PR to add the "missing" key?

Reply:
I'll just do it.